engine/src/sol/retry_rpc.rs

```diff
@@ -208,9 +209,34 @@ impl SolRetryRpcApi for SolRetryRpcClient {
 				Box::pin(move |client| {
 					let encoded_transaction = encoded_transaction.clone();
 					#[allow(clippy::redundant_async_block)]
-					Box::pin(
-						async move { client.send_transaction(encoded_transaction, config).await },
-					)
+					Box::pin(async move {
+						let tx_signature =
+							client.send_transaction(encoded_transaction, config).await?;
+
+						for status_retry in 0..GET_STATUS_BROADCAST_RETRIES {
+							let signature_statuses =
+								client.get_signature_statuses(&[tx_signature], true).await?;
+							match signature_statuses
+								.value
+								.first()
+								.and_then(|status| status.as_ref())
+							{
+								Some(_) => return Ok(tx_signature),
+								None => {
+									// Ignore sleep at last step
+									if status_retry < GET_STATUS_BROADCAST_RETRIES {
+										// Retry twice a second
```

Comment:
this comment is only as accurate if the constant doesn't change - would just remove the comment tbh, it's pretty clear